gobblin-cluster/src/main/java/gobblin/cluster/GobblinHelixTask.java

```diff
@@ -113,10 +111,15 @@ public TaskResult run() {
       Path workUnitFilePath =
           new Path(this.taskConfig.getConfigMap().get(GobblinClusterConfigurationKeys.WORK_UNIT_FILE_PATH));
 
-      WorkUnit workUnit =
-          workUnitFilePath.getName().endsWith(AbstractJobLauncher.MULTI_WORK_UNIT_FILE_EXTENSION) ? MultiWorkUnit
-              .createEmpty() : WorkUnit.createEmpty();
-      SerializationUtils.deserializeState(this.fs, workUnitFilePath, workUnit);
+      String fileName = workUnitFilePath.getName();
```

Comment:
Yes, we need to decide on some generic names, but I think this will require a broader discussion and more global cleanup. It seems like most hierarchical storage systems will be similar to a file system, so the file system terminology may mostly work. The current StateStore has the following terms: storeRootDir, stateStoreTableName, storeName, and tableName. TableName was probably meant to be a generic term instead of calling it fileName, but it is confusing for both FS and DB state stores. TableName refers to the last level of the key. FileName would be the closest match for FS and maybe itemName would be a generic equivalent.